data/json/vitrification/vitrified_items.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "id": "imperfect_doll_complete",
+    "type": "GENERIC",
+    "name": { "str": "imperfect doll" },
+    "description": "Its legs are mismatched, its arms mishapen, its eyes more crooked than its smile. Everywhere, it sparkles with the golden thread that has put it back together. It's beautiful.",
+    "category": "other",
+    "material": [ "cotton" ],
+    "price": 0,
+    "price_postapoc": 10000000,
+    "symbol": "@",
+    "color": "white",
+    "looks_like": "talking_doll",
+    "use_action": {
+      "type": "effect_on_conditions",
+      "description": "Hug the imperfect doll.",
+      "effect_on_conditions": [ "EOC_queue_unvitrified_return" ]
+    },
+    "flags": [ "SLEEP_AID", "NO_SALVAGE", "INTEGRATED", "ZERO_WEIGHT" ]
+  },
+  {
+    "id": "imperfect_doll_legs",
+    "type": "GENERIC",
+    "name": { "str": "floppy cotton legs" },
+    "description": "They're different colors and different lengths. It would be hard to run away with these.",
+    "category": "other",
+    "material": [ "cotton" ],
+    "price": 0,
+    "price_postapoc": 0,
+    "symbol": "i",
+    "color": "dark_gray",
+    "flags": [ "NO_SALVAGE", "INTEGRATED", "ZERO_WEIGHT" ]
+  },
+  {
+    "id": "imperfect_doll_arms",
+    "type": "GENERIC",
+    "name": { "str": "threadbare cotton arms" },
+    "description": "The spun cloth fingers are fraying away. There's nothing that they could hold on to.",
+    "category": "other",
+    "material": [ "cotton" ],
+    "price": 0,
+    "price_postapoc": 0,
+    "symbol": "i",
+    "color": "cyan",
+    "flags": [ "NO_SALVAGE", "INTEGRATED", "ZERO_WEIGHT" ]
+  },
+  {
+    "id": "imperfect_doll_features",
+    "type": "GENERIC",
+    "name": { "str": "buttons and thread" },
+    "description": "A pair of mismatched buttons and some dull yellow thread, to put on a brave face with.",
+    "category": "other",
+    "material": [ "cotton" ],
+    "price": 0,
+    "price_postapoc": 0,
+    "symbol": "i",
+    "color": "dark_gray",
+    "flags": [ "NO_SALVAGE", "INTEGRATED", "ZERO_WEIGHT" ]
+  },
+  {
+    "id": "glass_book",
+    "type": "GENERIC",
+    "name": { "str": "battered glass storybook" },
+    "description": "A black glass model of an old, tattered little book with a broken spine. To your surprise, it can be opened to a place in the middle. Inside, the pages are blank and perfectly reflective.",
```

Comment:
```suggestion
    "description": "A black glass model of an old, tattered little book with a broken spine.  To your surprise, it can be opened to a place in the middle.  Inside, the pages are blank and perfectly reflective.",
```